src/items/LinkItem.tsx

```diff
@@ -50,6 +58,7 @@ const LinkItem: FC<LinkItemProps> = ({
   openLinkMessage = 'Click here to open the link manually',
   height: defaultHeight,
   errorMessage = 'The link is malformed.',
+  isExtendable = false,
```

Comment:
So `isExtendable` is `isResizable`? I think I'm still confused but could we also match the naming with `resizing`? This comment applies to all other files.